shell/components/Resource/Detail/StatusRow.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import Bubble from '@shell/components/Resource/Detail/Card/PodsCard/Bubble.vue';
+import { StateColor, stateColorCssVar } from '@shell/utils/style';
+
+export interface Props {
+        color: StateColor;
+        label: string;
+        count: number;
+        percent: number;
+}
+
+const {
+  color, label, count, percent
+} = defineProps<Props>();
+</script>
+
+<template>
+  <div class="status-row">
+    <div
+      class="indicator"
+      :style="{backgroundColor: stateColorCssVar(color)}"
+    >
+&nbsp;
```

Comment:
Can we define our spacing via css instead of using `&nbsp`?

Reply:
We do actually, we have the `.indicator` class defined below.

I think this may have been a span before and I added the content so the element wouldn't collapse. 

I'll go ahead and remove it.